iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpView.swift

```diff
@@ -11,9 +11,13 @@ import Strings
 import Theming
 import Components
 
-struct SignUpView: View {
+public struct SignUpView: View {
 
-	var body: some View {
+	public init() {
+
+	}
```

Comment:
Can we move the } of this init up to the same line

Reply:
Done